db/version_edit.cc

```diff
@@ -375,6 +382,12 @@ const char* VersionEdit::DecodeNewFile4From(Slice* input) {
             }
           }
           break;
+        case kMinTimestamp:
+          f.min_timestamp = field.ToString();
+          break;
+        case kMaxTimestamp:
+          f.max_timestamp = field.ToString();
```

Comment:
Can we add a check for the equality of `min_timestamp.size()` and `max_timestamp.size()` somewhere in this method?

Reply:
sure, added